crates/module-system/hyperlane/tests/integration/with_agent/helpers/mod.rs

```diff
@@ -197,7 +197,7 @@ impl HyperlaneBuilder {
 
         // Current image is based on https://github.com/Sovereign-Labs/hyperlane-monorepo/tree/integration-2025-09-17-rebase branch
         let docker_image = docker_image
-            .unwrap_or_else(|_| "ghcr.io/sovereign-labs/hyperlane-agent:integration-5".into());
+            .unwrap_or_else(|_| "ghcr.io/ross-weir/hyperlane-agent:integration-6".into());
```

Comment:
will change back in follow up, I currently don't have perms to push to sov labs ghcr